full/src/Text/Interpolation/Nyan/Tutorial.hs

```diff
@@ -223,6 +223,10 @@ The quoter will return concrete t'Builder'.
 
 The quoter will return any type with t'FromBuilder' instance.
 
+==== c ([c]omments handling)
+
+Ignore line comments starting with @--@ and/or block comments enclosed in @{- ... -}@.
```

Comment:
I would use some different wording, as "ignore" may mean  "ignore it in processing, do not treat it in a special way" and so "leave it intact".

Reply:
Sure, I've decided to go for `Handle comments` and provide some clarifying examples. Hope it makes more sense now :)

BTW I've moved the lines upwards a little, now the section's before the ones about the return types.

Reply:
Oh that's actually good that you moved this section above, right.